### PR TITLE
Adds Education Section and College Score Card API

### PR DESCRIPTION
**Pull request by saykaren**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [ ] Your additions are ordered alphabetically
- [ ] Your submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column should be padded with one space on either side
- [ ] You have searched the repository for any relevant issues or pull requests
- [ ] Any category you are creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -492,6 +493,12 @@ API | Description | Auth | HTTPS | CORS |
 | [Represent by Open North](https://represent.opennorth.ca/) | Find Canadian Government Representatives | No | Yes | Unknown |
 | [USAspending.gov](https://api.usaspending.gov/) | US federal spending data | No | Yes | Unknown |
 
+**[⬆ Back to Index](#index)**
+### Education
+API | Description | Auth | HTTPS | CORS |
+|---|---|---|---|---|
+| [CollegeScoreCard.ed.gov](https://api.data.gov/ed/collegescorecard/v1/schools) | College Score Card | Yes | Yes | Unknown |
```
Comment (marekdano) [inline README.md:500]: This fits better for the `Open Data` section. Can you please move it there?

Creating a new section requires having a least 3 records there.

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for the contribution @saykaren!
Can you please review the requested change?

**Comment by leahfern** [comment]
This API has already been added to the list separately.

**Comment by matheusfelipeog** [comment]
This PR is inactive, so the API was added in another PR/commit.

More information at: #2002

**Pull request closed without merging**